### PR TITLE
Move eye_damage from /mob/ to eye organs

### PR DESCRIPTION
**Pull request by Muncher2112**
Moves /var/eye_damage from a var in /mob/mob_defines.dm to /modules/surgery/organs/eyes.dm

Changes instances that effected eye_damage to effect eye organs, using getorganslot("eyes_sight")

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Nice

File: code/modules/mob/living/carbon/status_procs.dm [hunk 1]
```diff
@@ -3,28 +3,37 @@
 // eye damage, eye_blind, eye_blurry, druggy, BLIND disability, NEARSIGHT disability, and HUSK disability.
 
 /mob/living/carbon/damage_eyes(amount)
+	var/obj/item/organ/eyes/eyes = src.getorganslot("eyes_sight")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/status_procs.dm]: implied `src`.

File: code/modules/mob/living/carbon/status_procs.dm [hunk 2]
```diff
@@ -3,28 +3,37 @@
...
+			if(eyes.eye_damage > 30)
 				overlay_fullscreen("eye_damage", /obj/screen/fullscreen/impaired, 2)
 			else
 				overlay_fullscreen("eye_damage", /obj/screen/fullscreen/impaired, 1)
 
 /mob/living/carbon/set_eye_damage(amount)
-	eye_damage = max(amount,0)
-	if(eye_damage > 20)
-		if(eye_damage > 30)
+	var/obj/item/organ/eyes/eyes = src.getorganslot("eyes_sight")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/status_procs.dm]: implied `src`.

File: code/modules/mob/living/carbon/status_procs.dm [hunk 3]
```diff
@@ -3,28 +3,37 @@
...
+	eyes.eye_damage = max(amount,0)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/status_procs.dm]: this should be below the `If(!eyes)` return` check.

File: code/modules/mob/living/carbon/status_procs.dm [hunk 4]
```diff
@@ -3,28 +3,37 @@
...
 		else
 			overlay_fullscreen("eye_damage", /obj/screen/fullscreen/impaired, 1)
 	else
 		clear_fullscreen("eye_damage")
 
 /mob/living/carbon/adjust_eye_damage(amount)
-	eye_damage = max(eye_damage+amount, 0)
-	if(eye_damage > 20)
-		if(eye_damage > 30)
+	var/obj/item/organ/eyes/eyes = src.getorganslot("eyes_sight")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/status_procs.dm]: implied `src`.

File: code/datums/diseases/advance/symptoms/vision.dm [hunk 5]
```diff
@@ -76,6 +79,7 @@ Bonus
 /datum/symptom/visionaid/Activate(datum/disease/advance/A)
 	..()
 	var/mob/living/M = A.affected_mob
+	var/obj/item/organ/eyes/eyes = M.getorganslot("eyes_sight")
```
Comment (Fox-McCloud) [inline code/datums/diseases/advance/symptoms/vision.dm]: need a 
```
if(!eyes)
 	return
```

here.

**Review by Cyberboss — APPROVED** [review]

**Comment by KorPhaeron** [comment]
@matthewvwalker 

Congratulations you can ignore the freeze for a few PRs now

**Pull request merged**